src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -294,9 +308,10 @@ namespace ts.server.typingsInstaller {
                             this.missingTypingsSet.set(packageName, true);
                             continue;
                         }
-                        if (!this.packageNameToTypingLocation.has(packageName)) {
-                            this.packageNameToTypingLocation.set(packageName, typingFile);
-                        }
+
+                        const newVersion = Semver.parse(this.typesRegistry.get(packageName)[`ts${ts.versionMajorMinor}`]);

```

Comment:
I'm worried about prerelease versions of TypeScript not finding themselves in the typings registry and getting `undefined` from this expression `this.typesRegistry.get(packageName)[`ts${ts.versionMajorMinor}`]`

Reply:
Defaulted to the `latest` tag if nonexistent version.